marklogic-data-hub-central/ui/e2e/cypress/integration/curation/load/defaultIngestion.spec.tsx

```diff
@@ -170,13 +170,48 @@ describe("Default ingestion ", () => {
     cy.uploadFile("input/test-1.json");
     cy.verifyStepRunResult("success", "Ingestion", stepName);
     tiles.closeRunMessage();
-    runPage.deleteStep(stepName).click();
-    loadPage.confirmationOptions("Yes").click();
 
-    //Delete the flow
+    //Verify Run Load step in flow where step exists, should run automatically
+    cy.waitUntil(() => toolbar.getLoadToolbarIcon()).click();
+    loadPage.loadView("table").click();
+    loadPage.runStep(stepName).click();
+    loadPage.runStepExistsOneFlowConfirmation().should("be.visible");
+    loadPage.confirmContinueRun();
+    cy.waitForAsyncRequest();
+    cy.verifyStepAddedToFlow("Load", stepName);
+    cy.uploadFile("input/test-1.json");
+    cy.verifyStepRunResult("success", "Ingestion", stepName);
+    tiles.closeRunMessage();
+
+    // add step to a new flow
+    cy.waitUntil(() => toolbar.getLoadToolbarIcon()).click();
+    loadPage.loadView("table").click();
+    loadPage.addStepToNewFlowListView(stepName);
+    cy.findByText("New Flow").should("be.visible");
+    runPage.setFlowName(flowName2);
+    runPage.setFlowDescription(`${flowName2} description`);
+    loadPage.confirmationOptions("Save").click();
+    cy.verifyStepAddedToFlow("Load", stepName);
+
+    //Verify Run Load step where step exists in multiple flows, choose one to automatically run in
+    cy.waitUntil(() => toolbar.getLoadToolbarIcon()).click();
+    loadPage.loadView("table").click();
+    loadPage.runStep(stepName).click();
+    loadPage.runStepExistsMultFlowsConfirmation().should("be.visible");
+    loadPage.selectFlowToRunIn(flowName);
+    cy.waitForAsyncRequest();
+    cy.verifyStepAddedToFlow("Load", stepName);
+    cy.uploadFile("input/test-1.json");
+    cy.verifyStepRunResult("success", "Ingestion", stepName);
+    tiles.closeRunMessage();
+
+    //Delete the flows
```

Comment:
@briantang Can we add a validation where when we delete the flow, the flow name should not be displayed in the list of flows for that step.

Reply:
@ngodugu-marklogic Sure, added and pushed validation in the spots where that scenario applies.